src/js/controller/AppController.js

```diff
@@ -129,7 +130,10 @@ L.Dropchop.AppController = L.Class.extend({
             return this._handleFormSubmit( opsConfig, e );
         } else {
             var form = this.forms.render( e.action, config );
-            form.on( 'submit', this._handleFormSubmit.bind( this, opsConfig ) );
+            if (!this.clicked) {
+                this.clicked = true;
+                form.on( 'submit', this._handleFormSubmit.bind( this, opsConfig ) );
+            }
```

Comment:
I think that this introduces a new bug:

1.  `this.clicked` starts off as `false`
1. I click the `bezier` icon, `this.clicked` turns to `true`, `submit` button becomes bound to the `geoOpsConfig`
1. I click a non-`geoOpsConfig` operation (eg add file). At this point, the `execute` button is still bound to the `geoOpsConfig` despite the fact that that I want a `fileOpsConfig` operation.

![new-bug](https://cloud.githubusercontent.com/assets/897290/9152395/6de23a04-3de3-11e5-855c-ce710a840995.gif)